src/components/pages/order/navbar/NavbarRightSide.js

```diff
@@ -1,11 +1,41 @@
-import { Link } from "react-router-dom";
+
 import styled from "styled-components";
+import ToggleButton from "../../../reusable-ui/ToggleButton";
 import Profile from "./Profile";
+import { useState } from "react"
+import ToastAdmin from "./ToastAdmin";
+import { toast } from "react-toastify";
+
 
 export default function NavbarRigthSide({username}) {
```

Comment:
Petit détail : Eventuellement appeler ton composant juste "RightSide" car en soit il "NavbarRightSide", est déjà dans un composant parent "NavBar".
Cela te ferais une architecture de composant plus clean et moins dense niveau naming je pense !

```suggestion
export default function RightSide({username}) {
```

Reply:
Ta props `username` que tu fais passer de composant en composant (OrderPage > Navbar > NavbarRightSide > Profile)
Tu peux directement la consommer là où tu en à besoin en l'occurrence dans ton composant `<Profile />`

Reply:
> Petit détail : Eventuellement appeler ton composant juste "RightSide" car en soit il "NavbarRightSide", est déjà dans un composant parent "NavBar". Cela te ferais une architecture de composant plus clean et moins dense niveau naming je pense !

C'est vrai, tu as raison @Ovgodd 💯

Reply:
> 

Je suis d'accord avec toi aussi @DavidRelut  💯